### DISCUSSION TITLE
legacy JS API in Dart Sass 2.0.

### DISCUSSION THREAD

**Discussion by mrwebg** [parcel-bundler/parcel#9957] [Q&A]
I am using Parcel to build my css and js. 
This is my package.json

`{
  "name": "",
  "version": "1.0.0",
  "description": "",
  "scripts": {
    "development": "parcel watch src/js/index.js --dist-dir ./dist",
    "production": "parcel build src/js/index.js --dist-dir ./dist",
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "",
  "license": "ISC",
  "browserslist": "defaults",
  "dependencies": {
    "bootstrap": "^5.3.3",
    "parcel": "^2.12.0"
  },
  "devDependencies": {
    "@parcel/transformer-sass": "^2.12.0"
  }
}`

This is the index.js 

`
import 'bootstrap';
import '../scss/main/main.scss';
`

When i run the script "development" in Powershell Terminal like:

`npm run development`

i get this warning:

```
⠇ Building index.js...
DEPRECATION WARNING: The legacy JS API is deprecated and will be removed in Dart Sass 2.0.0.
✨ Built in 1.70s

dist\index.js     78.79 KB    204ms
dist\index.css       123 B     71ms
```

Must i be worried to keep using Parcel for my builds? Can i solve this issue myself?

**Comment by otoinsa** [comment]
I have the same exact issue, how can this be resolved?

**Reply by devongovett** [reply]
See #9966
